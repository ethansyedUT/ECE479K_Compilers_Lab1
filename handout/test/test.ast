#5
_program
  #5
  _class
    Main
    Object
    "test.cl"
    (
    #4
    _method
      main
      Int
      #3
      _int
        0
      : Int
    )
