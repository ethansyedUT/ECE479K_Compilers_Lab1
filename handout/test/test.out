Main_main() returned 10
